### PR TITLE
made the footer in mui

### PR DESCRIPTION
**Pull request by simonpizevski**
image

We can change colors and links when we have the routes.

### CODE REVIEW

File: app/ui/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  Typography,
+  Link,
+  IconButton,
+  Divider,
+} from '@mui/material';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import InstagramIcon from '@mui/icons-material/Instagram';
+
+const theme = createTheme({
```
Comment (NelsonTheBrave) [inline app/ui/footer.tsx]: Looks nice! Just a question about the theme object you have inside the component. Seems like you define the palatte inside the component, just wondering if it is not supposed to be defined somewhere else to work more globally in the project? Is this a temporary fix? Please correct me if I'm wrong, I haven't worked with MUI components yet :)

Comment (simonpizevski) [reply app/ui/footer.tsx]: I also think that we can move this to another file in the project but i didnt know where for now so i put it here. Im gonna look into it!

Comment (JrSchoolAccount) [reply app/ui/footer.tsx]: You can remove the theme, the site already has a default dark theme and a themeProvider in app/layout.tsx
``` tsx
import { AppRouterCacheProvider } from '@mui/material-nextjs/v13-appRouter';
import type { Metadata } from 'next';
import { Inter } from "next/font/google";
import "./globals.css";
import ResponsiveAppBar from './ui/navbar';
import { ThemeProvider } from '@mui/material/styles';
import theme from './lib/theme'

const inter = Inter({ subsets: ["latin"] });

export const metadata: Metadata = {
  title: "Create Next App",
  description: "Generated by create next app",
};

export default function RootLayout({children}: { children: React.ReactNode }) {
  return (
    <html lang="en" className='dark'>
      <body>
        <AppRouterCacheProvider>
          <ThemeProvider theme={theme}>
          <ResponsiveAppBar />
          {children}
          </ThemeProvider>
        </AppRouterCacheProvider>
      </body>
    </html>
  );
}
```

Comment (JrSchoolAccount) [reply app/ui/footer.tsx]:
```tsx
<AppRouterCacheProvider>
          <ThemeProvider theme={theme}>
          <ResponsiveAppBar />
          {children}
PLACE THE FOOTER HERE <-----
          </ThemeProvider>
        </AppRouterCacheProvider>
        ```

**Comment by simonpizevski** [comment]
image
 updated footer

**Review by JrSchoolAccount — APPROVED** [review]
Looks good 👍

**Review by NelsonTheBrave — APPROVED** [review]
Good fixes

**Pull request merged**